scene/animation/animation_node_state_machine.cpp

```diff
@@ -1582,13 +1583,6 @@ void AnimationNodeStateMachine::remove_transition_by_index(const int p_transitio
 	Transition tr = transitions[p_transition];
 	transitions.write[p_transition].transition->disconnect("advance_condition_changed", callable_mp(this, &AnimationNodeStateMachine::_tree_changed));
 	transitions.remove_at(p_transition);
-
-	Vector<String> path_from = String(tr.from).split("/");
```

Comment:
You are removing this because it is unused? Shouldn't that be a separate PR?

Reply:
Agree, especially when I see how fast codestyle PRs are merged :)